troubleshoot-stale-read.md

```diff
@@ -100,6 +100,28 @@ The preceding output helps you determine:
 - Whether the apply index is too small to update safe-ts.
 - Whether the leader is sending a sufficiently updated resolved-ts when a follower peer exists.
 
+### Use logs to diagnose
+
+Every 10 seconds, TiKV checks the following metrics:
```

Comment:
How about linking the source code address here.

Reply:
I completely agree with @ekexium. Since this document is intended for troubleshooting, our primary focus should be on providing diagnostic methods rather than delving into specific implementation details. Therefore, it is not recommended to link source code here.